.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1.1.2
+      with:
+        ruby-version: 2.6
+    
+    - name: Install dependencies
+      run: bundle install
```

Comment:
I think we'll need one more step after this, to install `rpm` and (maybe?) `dpkg-deb` so we can actually build those packages. We have some docs on the packaging process:
- https://github.com/Shopify/shopify-app-cli/tree/master/packaging/debian
- https://github.com/Shopify/shopify-app-cli/tree/master/packaging/rpm

Reply:
I found from local testing that it wasn't necessary 🤨 but will keep an eye out for any issues surrounding this!

Reply:
Cool, those packages were needed for mac (obviously) so I expected `dpkg` to be present for ubuntu, but good to know that `rpm` is there by default too!